tests/ui/lang-items/required-lang-item.rs

```diff
@@ -1,10 +1,11 @@
+//@ add-core-stubs
 //@ edition: 2018
 
 #![feature(lang_items, no_core)]
 #![no_core]
 #![no_main]
 
-#[lang="copy"] pub trait Copy { }
-#[lang="sized"] pub trait Sized { }
+extern crate minicore;
+use minicore::*;
 
 async fn x() {} //~ ERROR requires `ResumeTy` lang_item
```

Comment:
Discussion: should this test use `minicore`, if it's actually trying to exercise (presumably) that:

1. As an example, `async fn x() {}` decl requires `ResumeTy` `lang_item` to be defined, and
2. Missing a required lang item emits an error-level diagnostics?

I'm inclinded to say that this test probably shouldn't be using `minicore`. Wdyt?

Reply:
Will change back to non-minicore.